riscv/src/large_field/runtime.rs

```diff
@@ -17,7 +17,7 @@ use crate::RuntimeLibs;
 #[derive(Clone)]
 pub struct Runtime {
     submachines: BTreeMap<String, SubMachine>,
-    syscalls: BTreeMap<Syscall, SyscallImpl>,
+    syscalls: BTreeMap<&'static str, SyscallImpl>,
```

Comment:
This will change the order the "jump table" will be emitted, but it is still deterministic.